freqtrade/misc.py

```diff
@@ -115,9 +115,9 @@ def throttle(func: Callable[..., Any], min_secs: float, *args, **kwargs) -> Any:
     :return: Any
     """
     start = time.time()
-    result = func(*args, **kwargs)
+    (result, extra_time)  = func(*args, **kwargs)
     end = time.time()
-    duration = max(min_secs - (end - start), 0.0)
+    duration = max(max(min_secs - (end - start), 0.0), extra_time)
```

Comment:
Why to not doing 
```
duration = max(min_secs - (end - start), extra_time)
```
It will avoid executing twice max (`max(max())`). Because your extra_time has already a default value at 0